### DISCUSSION TITLE
js in plausible.js on my own js files

### DISCUSSION THREAD

**Discussion by simonvdfr** [plausible/analytics#2568] [Anything Else]
do you think it is possible to put the line of JavaScript that is in plausible.js in our own js files, in order to have the least file load, for an even lighter application ?
I would like to include it in my cms to encourage even more people to use plausible :)

**Comment by jess1k** [comment]
You certainly can.
I'm not sure if you're aware, but the tracker is available as a [npm package](https://www.npmjs.com/package/plausible-tracker), which means you can easily include it in your bundled js app.
If you don't use a bundler, you can simply inline the script's code into your HTML or copy the script into your JS files; however, keep in mind that the publicly hosted script is wrapped in an iife, so you should consider copying the code from the build folder [here](https://www.npmjs.com/package/plausible-tracker?activeTab=explore).

**Reply by ukutaht** [reply]
Yes, it's certainly possible. If you inline the script itself, you won't get any automatic updates from us. But we will surely maintain backwards compatibility so it won't break in the future, and you can always refresh the inline code at any point in the future.
